<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicApplication</class>
 <widget class="QWidget" name="MusicApplication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1033</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天天酷音</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../TTKQrc/MusicPlayer.qrc">
    <normaloff>:/image/windowicon</normaloff>:/image/windowicon</iconset>
  </property>
  <widget class="QToolButton" name="windowClose">
   <property name="geometry">
    <rect>
     <x>977</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="minimization">
   <property name="geometry">
    <rect>
     <x>949</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>994</width>
     <height>620</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="musicPrevious">
   <property name="geometry">
    <rect>
     <x>64</x>
     <y>576</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicNext">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>576</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicKey">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>576</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="songsContainer">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>320</width>
     <height>490</height>
    </rect>
   </property>
  </widget>
  <widget class="MusicLocalSongSearchInlineEdit" name="musicSongSearchLine" native="true">
   <property name="geometry">
    <rect>
     <x>319</x>
     <y>32</y>
     <width>261</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>32</y>
     <width>25</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/share/searchlineleft);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowChangeSkin">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QStackedWidget" name="SurfaceStackedWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>115</y>
     <width>633</width>
     <height>455</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="MusicSongSearchOnlineWidget" name="songSearchWidget"/>
   <widget class="MusicLrcContainerForInline" name="musiclrccontainerforinline"/>
  </widget>
  <widget class="QPushButton" name="musicSearchWidgetButton">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>82</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>音乐</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicLrcWidgetButton">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>82</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>歌词</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchRefreshButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>82</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="musicIndexWidgetButton">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>82</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>33</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="userWindow">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QToolButton" name="musicButton_tools">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::InstantPopup</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowRemote">
   <property name="geometry">
    <rect>
     <x>918</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_playlist">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_radio">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_mydownl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_cloud">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="videoWidgetButton">
   <property name="geometry">
    <rect>
     <x>855</x>
     <y>82</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>视频</string>
   </property>
  </widget>
  <widget class="QToolButton" name="musicWindowConcise">
   <property name="geometry">
    <rect>
     <x>888</x>
     <y>30</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="resizeWindowLabel">
   <property name="geometry">
    <rect>
     <x>993</x>
     <y>620</y>
     <width>15</width>
     <height>15</height>
    </rect>
   </property>
   <property name="pixmap">
    <pixmap resource="../../../TTKQrc/MusicPlayer.qrc">:/image/resizeWindow</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="musicButton_mobile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="lrcDisplayAllButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>15</width>
     <height>56</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border:none;background:transparent;};</string>
   </property>
  </widget>
  <widget class="QPushButton" name="musicSearchBackButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>82</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>582</y>
     <width>46</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="musicQualityWindow"/>
  </widget>
  <widget class="MusicMarqueeWidget" name="showCurrentSong" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>578</y>
     <width>215</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;font-weight:bold;font-size:15px;</string>
   </property>
   <property name="text" stdset="0">
    <string>当前无正在播放的歌曲</string>
   </property>
  </widget>
  <widget class="MusicTimerSliderWidget" name="musicTimeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>593</y>
     <width>375</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="playCurrentTime">
   <property name="geometry">
    <rect>
     <x>532</x>
     <y>588</y>
     <width>35</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="playTotalTime">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>588</y>
     <width>37</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#FFFFFF;</string>
   </property>
   <property name="text">
    <string>/00:00</string>
   </property>
  </widget>
  <widget class="QToolButton" name="musicBestLove">
   <property name="geometry">
    <rect>
     <x>695</x>
     <y>598</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicPlayMode">
   <property name="geometry">
    <rect>
     <x>735</x>
     <y>595</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="musicDesktopLrc">
   <property name="geometry">
    <rect>
     <x>857</x>
     <y>597</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="MusicEnhancedWidget" name="musicEnhancedButton" native="true">
   <property name="geometry">
    <rect>
     <x>625</x>
     <y>600</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QSlider" name="musicSoundSlider">
   <property name="geometry">
    <rect>
     <x>915</x>
     <y>599</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSound">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>598</y>
     <width>18</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/image/soundmax);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicDownload">
   <property name="geometry">
    <rect>
     <x>815</x>
     <y>595</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="musicSimilarFound">
   <property name="geometry">
    <rect>
     <x>775</x>
     <y>595</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>musicPrevious</zorder>
  <zorder>musicNext</zorder>
  <zorder>musicKey</zorder>
  <zorder>songsContainer</zorder>
  <zorder>musicSongSearchLine</zorder>
  <zorder>musicSearchButton</zorder>
  <zorder>musicWindowChangeSkin</zorder>
  <zorder>SurfaceStackedWidget</zorder>
  <zorder>musicSearchWidgetButton</zorder>
  <zorder>musicLrcWidgetButton</zorder>
  <zorder>musicSearchRefreshButton</zorder>
  <zorder>musicIndexWidgetButton</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>musicButton_tools</zorder>
  <zorder>musicWindowRemote</zorder>
  <zorder>musicButton_playlist</zorder>
  <zorder>musicButton_radio</zorder>
  <zorder>musicButton_mydownl</zorder>
  <zorder>musicButton_cloud</zorder>
  <zorder>videoWidgetButton</zorder>
  <zorder>musicWindowConcise</zorder>
  <zorder>resizeWindowLabel</zorder>
  <zorder>minimization</zorder>
  <zorder>musicButton_mobile</zorder>
  <zorder>lrcDisplayAllButton</zorder>
  <zorder>windowClose</zorder>
  <zorder>musicSearchBackButton</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>showCurrentSong</zorder>
  <zorder>musicTimeWidget</zorder>
  <zorder>playCurrentTime</zorder>
  <zorder>playTotalTime</zorder>
  <zorder>musicBestLove</zorder>
  <zorder>musicPlayMode</zorder>
  <zorder>musicDesktopLrc</zorder>
  <zorder>musicEnhancedButton</zorder>
  <zorder>musicSoundSlider</zorder>
  <zorder>musicSound</zorder>
  <zorder>musicDownload</zorder>
  <zorder>musicSimilarFound</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MusicMarqueeWidget</class>
   <extends>QWidget</extends>
   <header>musicmarqueewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicLrcContainerForInline</class>
   <extends>QWidget</extends>
   <header>musiclrccontainerforinline.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicSongSearchOnlineWidget</class>
   <extends>QWidget</extends>
   <header>musicsongsearchonlinewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicTimerSliderWidget</class>
   <extends>QWidget</extends>
   <header>musictimersliderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicLocalSongSearchInlineEdit</class>
   <extends>QWidget</extends>
   <header>musiclocalsongsearchinlineedit.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MusicEnhancedWidget</class>
   <extends>QWidget</extends>
   <header>musicenhancedwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../TTKQrc/MusicPlayer.qrc"/>
 </resources>
 <connections/>
</ui>
